.github/build-with-maven-native.sh

```diff
@@ -71,5 +72,21 @@ credential=$(az appconfig credential list \
 export QUARKUS_AZURE_APP_CONFIGURATION_ID=$(echo "${credential}" | jq -r '.id')
 export QUARKUS_AZURE_APP_CONFIGURATION_SECRET=$(echo "${credential}" | jq -r '.value')
 
+az keyvault create \
+    --name ${KEY_VAULT_NAME} \
+    --resource-group ${RESOURCE_GROUP_NAME} \
+    --location eastus
```

Comment:
Mention this duplicates code in `integration-tests/README.md`.

Reply:
See https://github.com/quarkiverse/quarkus-azure-services/commit/5fce42ce138dd83c3e4baf06b1a1b76ec8c82829.